source/content/guides/decoupled-sites/04-site.md

```diff
@@ -109,18 +111,20 @@ Create a site without configuring the backend and link to a frontend framework.
 
 1. Select a Git provider from the **Choose your Git provider** options and click **Continue**.
 
-1. Select your desired GitHub account from the GitHub Account dropdown.
+1. Select your desired account from the GitHub Account dropdown.
 
 1. Choose the repository name from the Select Repository dropdown and click **Continue**.
+    ![A screenshot of the Dashboard showing a GitHub account and Repository options](../../../images/guides/decoupled-sites/import-repository.png)
 
 1. Add to the General Info section.
 
     * Specify the Site Name: The site name is the title of your site. You can edit the site name in **Settings** after creation. 
-    * Select the frontend framework. You can choose either Next.js or Gatsby as the static site generator.
+    * Select the frontend framework. You can choose either Next.js or Gatsby as the static site generator. 
+        If you are testing out a static React.js application, you may try selecting Gatsby. If the application is using the Node runtime then you will want to select Next.js as the framework. Please note we will not be able to provide support for frameworks outside of Next.js or Gatsby.
 
 1. (Optional) Link your CMS.
 
-    > **NOTE**: If you are using a specified CMS, you can link your CMS backend and the site environment from which to source content. However, this is more applicable when using the Decoupled starter kit templates.
+    > **NOTE**: If you are using a back-end CMS on Pantheon, you can link your CMS and the site environment from which to source content. However, this is optional and more applicable when using the Decoupled starter kit templates.
```

Comment:
I have some quibbles with this section that I have been struggling with.  From an EA standpoint, we definitely have customers who are potentially bringing sites over that are already configured to address a CMS.  I think we need to expand on documentation for that use case.

Reply:
That makes sense, I've just expanded this section in [dad37e8](https://github.com/pantheon-systems/documentation/pull/7715/commits/dad37e832fdbb14cd74760037b70ddc804df63d7)